hokusai/staging.yml

```diff
@@ -1,3 +1,4 @@
+----
```

Comment:
Same - from https://yaml.org/spec/1.0/
> YAML uses three dashes (“---”) to separate documents within a stream.

Reply:
i just removed both of them, which is what my linter prefers